src/Core/Resolvers/PostgresQueryBuilder.cs

```diff
@@ -228,7 +228,7 @@ public string BuildStoredProcedureResultDetailsQuery(string databaseObjectName)
         /// <inheritdoc/>
         public string BuildQueryToGetReadOnlyColumns(string schemaParamName, string tableParamName)
         {
-            string query = "SELECT attname AS column_name FROM pg_attribute " +
+            string query = "SELECT attname AS \"COLUMN_NAME\" FROM pg_attribute " +
```

Comment:
Are quotes needed here? The other statements don't seem to make use of double quotes, for example
![image](https://github.com/user-attachments/assets/8e7364e6-9194-4e65-94c0-75f269a141cf)

For style consistency would be better if we can remove these, or if quotes are needed see if single quote will work since we already use some single quotes in queries in this class.